src/sentry/static/sentry/app/views/eventsV2/eventQueryParams.tsx

```diff
@@ -0,0 +1,93 @@
+export type ColumnValueType = 'numeric' | 'string' | 'unknown';
+
+// todo(leedongwei): Find the correct types
+export const AGGREGATIONS = {
+  latest_event: {
+    type: 'string',
+    isSortable: true,
+  },
+  last_seen: {
+    type: 'string',
+    isSortable: true,
+  },
+  count: {
+    type: 'numeric',
+    isSortable: true,
+  },
+  count_unique: {
+    type: 'numeric',
+    isSortable: true,
+  },
+  rpm: {
+    type: 'numeric',
+    isSortable: true,
+  },
+  pXX: {
+    type: 'numeric',
+    isSortable: true,
```

Comment:
I wouldn't add `rpm` or `pxx` just yet, they aren't hooked up in the backend.

Reply:
Commented out `pXX` and `rpm`. I removed `avg` too based on [this comment](https://github.com/getsentry/sentry/blob/93f42fea2efcf2893389005bbf8b8222439c9bf5/src/sentry/api/event_search.py#L673).